lib/engine/game/g_18_zoo/meta.rb

```diff
@@ -8,7 +8,7 @@ module G18ZOO
       module Meta
         include Game::Meta
 
-        DEV_STAGE = :prealpha
+        DEV_STAGE = :alpha
```

Comment:
Please separate the 'alpha' change from this PR

Reply:
done in 1b2db46